packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.tsx

```diff
@@ -69,6 +69,7 @@ type QueryWrapperPlotNavOwnProps = {
   >
   facetsToPlot?: string[]
   availableFacets?: FacetFilterControlsProps['availableFacets']
+  facetValueSortConfigs?: FacetFilterControlsProps['facetValueSortConfigs']
```

Comment:
Could this name imply that the facet data is sorted in other places? The only other place I can think of is the plots (i.e. pie plot slices are ordered by sort rather than size). Consider a rename or comment to clarify that this is not the case, but maybe not a big deal.

Reply:
Added a lengthy comment for now